app/scripts/modules/core/src/application/applications.html

```diff
@@ -20,7 +20,7 @@ <h2 class="header-section">
   </div>
   <div class="container">
     <h2 ng-if="!$ctrl.applicationsLoaded" class="text-center" style="margin-bottom: 250px">
-      <span class="fa fa-cog fa-3x fa-spin"></span>
+      <loading-spinner size="'small'"></loading-spinner>
```

Comment:
This should be bigger than `small` (replacing `fa-3x`)

Reply:
done and tested.